packages/protocol/test/governance/voting/lockedgold.ts

```diff
@@ -771,4 +798,753 @@ contract('LockedGold', (accounts: string[]) => {
+                delegatedAmount
+              )
+            })
+
+            describe('When locked more gold and redelagate', () => {
+              let resp2: Truffle.TransactionResponse
+              beforeEach(async () => {
+                await lockedGold.lock({ value: value, from: delegator })
+                resp2 = await lockedGold.delegateGovernanceVotes(delegatee1, percentsToDelegate)
+              })
+
+              it('should delegate votes correctly', async () => {
+                assertEqualBN(
+                  await lockedGold.getAccountTotalGovernanceVotingPower(delegatee1),
+                  delegatedAmount * 2
+                )
+                await assertDelegatorDelegateeAmounts(
+                  delegator,
+                  delegatee1,
+                  percentsToDelegate,
+                  delegatedAmount * 2
+                )
+              })
+
+              it('should emit the CeloDelegated event', async () => {
+                assert.equal(resp2.logs.length, 1)
+                const log = resp2.logs[0]
+                assertLogMatches2(log, {
+                  event: 'CeloDelegated',
+                  args: {
+                    delegator: delegator,
+                    delegatee: delegatee1,
+                    percent: percentsToDelegate,
+                    amount: delegatedAmount * 2,
+                  },
+                })
+              })
+            })
+          })
+
+          describe('When 2 delegators are delegating to delegatee1', () => {
+            beforeEach(async () => {
+              await lockedGold.delegateGovernanceVotes(delegatee1, percentsToDelegate, {
+                from: delegator,
+              })
+              await lockedGold.delegateGovernanceVotes(delegatee1, percentsToDelegate, {
+                from: delegator2,
+              })
+            })
+
+            it('should delegate votes correctly', async () => {
+              assertEqualBN(
+                await lockedGold.getAccountTotalGovernanceVotingPower(delegatee1),
+                delegatedAmount * 2
+              )
+              await assertDelegatorDelegateeAmounts(
+                delegator,
+                delegatee1,
+                percentsToDelegate,
+                delegatedAmount
+              )
+
+              await assertDelegatorDelegateeAmounts(
+                delegator2,
+                delegatee1,
+                percentsToDelegate,
+                delegatedAmount
+              )
+            })
+          })
+        })
+      })
+    })
+  })
+
+  async function assertDelegatorDelegateeAmounts(
+    delegator: string,
+    delegatee: string,
+    percent: number,
+    amount: number
+  ) {
+    const [percentage, currentAmount] = await lockedGold.getDelegatorDelegateeInfo(
+      delegator,
+      delegatee
+    )
+    assertEqualBN(percentage, percent)
+    assertEqualBN(currentAmount, amount)
+  }
+
+  describe('#revokeDelegatedGovernanceVotes()', () => {
```

Comment:
2 tests "should revoke votes correctly when delegatee voting" are testing this scenario and since updateDelegatedAmount is currently already in revokeDelegatedGovernanceVotes it is not necessary to differentiate